docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -17,12 +17,20 @@ To explain the allocation of a shard, issue a request:
 
 [source,js]
 --------------------------------------------------
-$ curl -XGET 'http://localhost:9200/_cluster/allocation/explain' -d '{
+PUT /myindex
```

Comment:
I'd either call this out in the prose like the painless docs do for their example or I'd surround this in block comments (`////`) with an explanation that it is hidden but makes the snippets on the page make more sense.